### ISSUE TITLE
Disable word wrap in the editor only?

### ISSUE THREAD

**Issue by Forum1877621** [pbek/QOwnNotes#2566]
I would like to disable word wrap in the editor only, not the preview. Is this possible?

#### Expected behaviour

Word wrap is disabled in the editor only

#### Actual behaviour

Word wrap is enabled/disabled on both the editor and the preview

#### Output from the debug section in the settings dialog

<details><summary>Expand</summary>

</details>

#### Relevant log output in the Log panel

<details><summary>Expand</summary>

</details>

**Comment by pbek** [comment]
Just press `Ctrl + Shift + A` to search for actions in the main menu and search for `wrap`.

**Comment by pbek** [comment]
Hm, but that sets the wrap for both as you mentioned told me `MainWindow::initEditorSoftWrap()`. So the answer is "no, sorry". And I don't think you can override that behavior with a CSS style in the preview html.

**Comment by Forum1877621** [comment]
Would it be possible to add this as a feature?

**Comment by Forum1877621** [comment]
One reason I need word wrap only on the preview is that with tables with long sentences the preview shows them fine but the editor gets messed up by wrapping

example:

![image](https://user-images.githubusercontent.com/109703756/180658440-ba651cce-ac44-4523-9e7c-a47ff0969bc0.png)

**Comment by pbek** [comment]
But long tables just look fine with wordwrap disabled too...
I don't really want to add more complexity to that already very complex menu in adding one more conditional setting.

For your use-case you basically would just need to remove this line and build QOwnNotes yourself:
<https://github.com/pbek/QOwnNotes/blob/6cdcc868e110d1c2e1469e066c9f7e30e3037813/src/mainwindow.cpp#L1098>

Or set it to the desired value (`QTextEdit::WidgetWidth` or `QTextEdit::NoWrap`).

**Comment by Forum1877621** [comment]
> But long tables just look fine with wordwrap disabled too...

Yes, but the problem (especially with laptop screens) is that without word wrap on the preview I have to scroll to the right which takes too much time especially when I have a lot of notes which each have a long table in them

This makes it too difficult to edit and preview tables at the same time so currently I use VS Code to edit the table text and then go back to QOwnNotes to view the results

**Comment by pbek** [comment]
## 26.4.1

- Updated menu action text `Use softwrap in note editor and preview` to
  `Use softwrap in note editor`, since the QLiteHtml preview is always softwrapped
  for [#2566](https://github.com/pbek/QOwnNotes/issues/2566))
